packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -571,4 +562,46 @@ export class ProfilesUtils {
         }
         throw new Error(vscode.l10n.t("Tree Item is not a Zowe Explorer item."));
     }
+
+    private static async convertV1Profs(): Promise<void> {
```

Comment:
Noticed the logic for building a response message after converting v1 profiles has moved from ProfilesCache in ZE API to here. Do we think that extenders who wish to convert profiles don't need this code and plan to leave it up to them to process the response themselves?

Reply:
I believe we should keep the original response in the API call return so Extenders can manipulate the information how they wish.